python/lsst/obs/subaru/objectMasks.py

```diff
@@ -0,0 +1,132 @@
+import re
+import lsst.daf.base as dafBase
+import lsst.pex.logging as pexLog
+import lsst.afw.coord as afwCoord
+import lsst.afw.geom as afwGeom
+import lsst.afw.table as afwTable
+
+class ObjectMaskCatalog(object):
+    """Class to support bright object masks
+
+    N.b. I/O is done by providing a readFits method which fools the (old) butler.
+    """
+    def __init__(self):
+        schema = afwTable.SimpleTable.makeMinimalSchema()
+
+        schema.addField("radius", 'Angle', "radius of mask")
+        self._catalog = afwTable.SimpleCatalog(schema)
+        self._catalog.table.setMetadata(dafBase.PropertyList())
+
+        self.table  = self._catalog.table
+        self.addNew = self._catalog.addNew
+
+    def __len__(self):
+        return len(self._catalog)
+
+    def __iter__(self):
+        return iter(self._catalog)
+
+    def __getitem__(self, i):
+        return self._catalog.__getitem__(i)
+
+    def __setitem__(self, i, v):
+        return self._catalog.__setitem__(i, v)
+
+    @staticmethod
+    def readFits(fileName, hdu=0, flags=0):
+        """Read a ds9 region file, returning a ObjectMaskCatalog object
+
+        This method is called "readFits" to fool the butler. The corresponding mapper entry looks like
```

Comment:
Oh I see! Please file a (low-priority) issue for @n8pease and the Data Access team to look into some rainy afternoon.